### PR TITLE
Implementing stress testing with new relic and artillery

### PR DESCRIPTION
**Pull request by justindchan**

### CODE REVIEW

**Comment by justindchan** [comment]
@rfagg805 @pammyla @Thefirmt wheel of pull request

File: newrelic.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+'use strict'
```
Comment (inyhsu) [inline newrelic.js:1]: I think this file should be in gitignore, it has license_key

**Review by inyhsu — CHANGES_REQUESTED** [review]
newrelice.js should be in gitignore or you want to take off the license_key

File: newrelic.js (hunk 1 above)
Comment (justindchan) [reply newrelic.js:1]: adding to git ignore! @rfagg805

**Pull request closed without merging**